iothub/device/src/Transport/AmqpIot/AmqpIotSendingLink.cs

```diff
@@ -29,6 +29,15 @@ public AmqpIotSendingLink(SendingAmqpLink sendingAmqpLink)
             _sendingAmqpLink.Closed += SendingAmqpLinkClosed;
         }
 
+        /// <summary>
+        /// Removes all events that would have run in the Closed event. This value cannot be set to null
+        /// outside of this class due to how .NET events work.
+        /// </summary>
+        public void ClearClosedEventHandler()
+        {
+            Closed = null;
```

Comment:
I'm concerned about a possible memory leak here. Let's chat over Teams when we're both free.

Reply:
```suggestion
            Closed = null;
            _sendingAmqpLink.Closed -= SendingAmqpLinkClosed
```